test/cases/ostree-simplified-installer.sh

```diff
@@ -25,6 +25,11 @@ function get_build_info() {
 # Start firewalld
 sudo systemctl enable --now firewalld
 
+# Install fdo packages (This cannot be done in the spec file because Fedora repo does not have this package yet)
+greenprint "Install and start fdo-aio service"
+sudo dnf install -y fdo-admin-cli
+sudo systemctl enable --now fdo-aio
```

Comment:
it's really difficult to understand what's going on, I'd first try and grab the logs of the fdo-aio service for more info (also checking it's started successfully)